modelarmor/api/ModelArmor.Samples.Tests/ModelArmorFixture.cs

```diff
@@ -15,53 +15,180 @@
 */
 
 using System;
+using System.Collections.Generic;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
 using Google.Cloud.ModelArmor.V1;
 using Xunit;
 
 [CollectionDefinition(nameof(ModelArmorFixture))]
 public class ModelArmorFixture : IDisposable, ICollectionFixture<ModelArmorFixture>
 {
+    // Environment variable names.
+    private const string EnvProjectId = "GOOGLE_PROJECT_ID";
+    private const string EnvLocation = "GOOGLE_CLOUD_LOCATION";
+
     public ModelArmorClient Client { get; }
+    public DlpServiceClient DlpClient { get; }
     public string ProjectId { get; }
     public string LocationId { get; }
-    public TemplateName TemplateForQuickstartName { get; }
+    private readonly List<TemplateName> _maTemplatesToCleanup = new List<TemplateName>();
+    private readonly List<string> _dlpTemplatesToCleanup = new List<string>();
 
     public ModelArmorFixture()
     {
-        // Get the Google Cloud Project ID.
-        ProjectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
-        if (string.IsNullOrEmpty(ProjectId))
+        ProjectId = GetRequiredEnvVar(EnvProjectId);
+        LocationId = Environment.GetEnvironmentVariable(EnvLocation) ?? "us-central1";
+
+        // Create the Model Armor client.
+        ModelArmorClientBuilder clientBuilder = new ModelArmorClientBuilder
+        {
+            Endpoint = $"modelarmor.{LocationId}.rep.googleapis.com",
+        };
+        Client = clientBuilder.Build();
```

Comment:
Tiny nit, but for consistency, can you create this client in a single line similar to the DLP one?